### ISSUE TITLE
"git lfs uninstall" does nothing.

### ISSUE THREAD

**Issue by axkibe** [git-lfs/git-lfs#5588]
In my tries to reproduce this "should have been a pointer" things, I came to this:

```
atest@axel:~$ git lfs uninstall
atest@axel:~$ git clone ssh://axel@git.csc.univie.ac.at/axeltest.git
Cloning into 'axeltest'...
remote: Enumerating objects: 41, done.
remote: Counting objects: 100% (12/12), done.
remote: Compressing objects: 100% (12/12), done.
remote: Total 41 (delta 3), reused 0 (delta 0), pack-reused 29
Receiving objects: 100% (41/41), 111.68 KiB | 15.95 MiB/s, done.
Resolving deltas: 100% (12/12), done.
Encountered 1 file that should have been a pointer, but wasn't:
	zeal_zeal.png
```

git lfs uninstall doesnt do anything?

the filters are still active.

debian bookworm
git-lfs/3.3.0 (GitHub; linux amd64; go 1.19.8)

I guess debian puts the somewhere global?

**Comment by axkibe** [comment]
I mean yes of course we cannot change here debian global system settings, but IMO both install and uninstall should complain/warn if system-wide filters are already active.

**Comment by bk2204** [comment]
I think we can definitely add a check here if, after the `uninstall`, the filters are still active.  Let me mark this accordingly.

**Comment by axkibe** [comment]
I agree, same should we maybe deny an install if its already system installed?

BTW: the debian package has in the postinst skript:
```
git lfs install --skip-repo --system > /dev/null 2>&1
```
Thats why. was just confusing. I ran "sudo git lfs uninstall --skip-repo --system" on my machine so for testing I can turn it on/off on user level.
